sesman/Makefile.am

```diff
@@ -61,6 +61,8 @@ xrdp_sesman_SOURCES = \
   sesman.h \
   session.c \
   session.h \
+  sessionrecord.c \
```

Comment:
I don't like the name `sessionrecord.c` 
its misleading, one might think of graphical session recording.
can you think of a better name? maybe `logontrack.c` or just `utmp.c`?

Reply:
First work used utmp.c and utmp.h but the #include directive was not using the first .h (with <> or "")